frontend/resources/translations/it-IT.json

```diff
@@ -172,17 +173,17 @@
     "menu": {
       "appearance": "Aspetto",
       "language": "Lingua",
-      "open": "aprire le impostazioni",
+      "open": "Apri le impostazioni",
       "user": "Utente",
       "files": "File",
       "supabaseSetting": "Impostazione Supabase"
     },
     "appearance": {
       "themeMode": {
-        "label": "Theme Mode",
+        "label": "Tema",
```

Comment:
we already use Tema for the word "Theme", is there something more appropriate for "Theme Mode"?

Reply:
`Luminosità del tema` (Theme brightness) or `Modalità tema` (Theme mode)

Maybe one of these? I'm no italian though, but google uses the second one, modalità tema in one of their support answers: 
https://support.google.com/accessibility/android/answer/6151800?hl=it

Reply:
Hi, as an Italian myself I felt "Tema" to be good, but "Modalità tema" is good as well, just updated the string to the latter.